### PR TITLE
add truffle CLI debugger customizing variables view description

### PR DESCRIPTION
**Pull request by dongmingh**
The truffle CLI debugger supports customizing variables view
(see truffle issue #4360).  The reference document needs to be
updated accordingly.

### CODE REVIEW

**Review by haltman-at — COMMENTED** [review]
OK, this is much better!  Just a few minor things I think should be fixed, then I think this is good.

File: src/docs/truffle/getting-started/debugging-your-contracts.md
Comment (haltman-at) [inline src/docs/truffle/getting-started/debugging-your-contracts.md:258]: Some minor clarity and grammar suggestions:

1. "You can display them" -> "You can still display them"
2. "will make" -> "will cause"
3. "will always be displayed" -> "will once again always be displayed"

You can probably delete the clause "and `v -bui` will turn this off", but if you think it's worth keeping, I'd suggest changing "will turn this off" to "will turn this off again".

**Review by haltman-at — COMMENTED** [review]
Sorry, here's the review comment I meant to include.

File: src/docs/truffle/getting-started/debugging-your-contracts.md
Comment (dongmingh) [reply src/docs/truffle/getting-started/debugging-your-contracts.md]: changed according to the comments.

**Review by haltman-at — APPROVED** [review]
Seems good to me!  Although obviously you should hold off on merging this until the corresponding Truffle PR is merged.  Thanks Dongming!

**Pull request merged**
